model_analyzer/triton/server/server_docker.py

```diff
@@ -115,7 +116,7 @@ def start(self):
         # Run the command in the container
         cmd = 'tritonserver ' + self._server_config.to_cli_string()
 
-        self._tritonserver_log = \
+        _, self._tritonserver_log_gen = \
             self._tritonserver_container.exec_run(cmd, stream=True)
```

Comment:
We can also include the stderr too. This may help us debug some of the intermittent errors we see in the CI.

Reply:
According to [this](https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.exec_run) I think that the bytestring includes both stdout and stderr? I could be wrong.

Reply:
I think you need to set the `stderr` to True. https://docker-py.readthedocs.io/en/stable/containers.html

Reply:
Sorry, it seems to be ok.